jest.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   transform: {
     '^.+\\.(t|j)sx?$': '@swc/jest',
   },
+  maxWorkers: process.env.CI ? 2 : undefined,
```

Comment:
Setting this here as we can no longer pass the flag to `pnpm test` anymore because that script now runs multiple other scripts.